warehouse/query-core/src/test/java/datawave/query/MiscQueryTest.java

```diff
@@ -96,6 +97,57 @@ public void testEventThreshold() throws Exception {
         runTest(query, expect);
     }
 
+    @Test(expected = InvalidQueryException.class)
+    public void testFieldIgnoreParam1() throws Exception {
+        log.info("------  testEventThreshold  ------");
+        // setting event per day does not alter results
+        this.logic.setEventPerDayThreshold(1);
+        String phrase = RE_OP + "'.*a'" + "&& FOO == bar2";
+        String query = Constants.ANY_FIELD + phrase + "&& FOO == bar2";
+        String expect = this.dataManager.convertAnyField(phrase);
+
+        Map<String,String> options = new HashMap<>();
+
+        // this will throw an exception due to the nonexistent fields not being ignored.
+        options.put(QueryParameters.IGNORE_NONEXISTENT_FIELDS, "false");
+
+        runTest(query, expect, options);
+    }
+
+    @Test
+    public void testFieldIgnoreParam2() throws Exception {
+        log.info("------  testEventThreshold  ------");
+        // setting event per day does not alter results
+        this.logic.setEventPerDayThreshold(1);
+        String phrase = RE_OP + "'.*a'" + "&& FOO == bar2";
+        String query = Constants.ANY_FIELD + phrase + "&& FOO == bar2";
+        String expect = this.dataManager.convertAnyField(phrase);
+
+        Map<String,String> options = new HashMap<>();
+
+        // this should allow the query to run successfully.
+        options.put(QueryParameters.IGNORE_NONEXISTENT_FIELDS, "true");
+
+        runTest(query, expect, options);
+    }
+
+    @Test
+    public void testFieldIgnoreParam3() throws Exception {
+        log.info("------  testEventThreshold  ------");
+        // setting event per day does not alter results
+        this.logic.setEventPerDayThreshold(1);
+        String phrase = RE_OP + "'.*a' && STATE == 'sta'";
+        String query = Constants.ANY_FIELD + phrase + "&& STATE == 'sta'";
+        String expect = this.dataManager.convertAnyField(phrase);
+
+        Map<String,String> options = new HashMap<>();
+
+        // this should allow the query to run successfully.
```

Comment:
This may be a copy and paste error on this comment

Reply:
Will remove from the latest commit